test/e2e/multihoming.go

```diff
@@ -792,6 +792,68 @@ var _ = Describe("Multi Homing", func() {
 						Expect(connectToServer(clientPodConfig, underlayServiceIP, servicePort)).To(MatchError(ContainSubstring("Connection timeout")))
 					})
 				})
+
+				When("an ingress deny-all policy is provisioned", func() {
+					var clientPodConfig podConfiguration
+					allPodsSelector := map[string]string{}
+
+					const denyAllIngress = "deny-all-ingress"
+
+					BeforeEach(func() {
+						clientPodConfig = podConfiguration{
+							name:        clientPodName,
+							namespace:   f.Namespace.Name,
+							attachments: []nadapi.NetworkSelectionElement{{Name: secondaryNetworkName}},
+							labels:      allPodsSelector,
+						}
+						kickstartPod(cs, clientPodConfig)
+					})
+
+					DescribeTable("can communicate over a localnet secondary network from pod to gw", func(mnp *mnpapi.MultiNetworkPolicy) {
+						By("provisioning the *deny-all* multi-network policy")
+						_, err := mnpClient.MultiNetworkPolicies(clientPodConfig.namespace).Create(
+							context.Background(),
+							mnp,
+							metav1.CreateOptions{},
+						)
+						Expect(err).NotTo(HaveOccurred())
+
+						By("asserting the *ingress deny-all* multi-net policy has no impact on egress")
+						Expect(connectToServer(clientPodConfig, underlayServiceIP, servicePort)).To(Succeed())
```

Comment:
Maybe we also need to make sure ingress is blocked (unless we have other test that does it already and then we can merge the tests)

Anyhow will take it from here as we spoke, just for visbility

Thanks